packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -323,6 +338,24 @@ class SliderThemeData extends Diagnosticable {
   /// The color given to the [valueIndicatorShape] to draw itself with.
   final Color valueIndicatorColor;
 
+  /// The shape and behavior that will be used to draw the [Slider]'s track.
+  ///
+  /// This can be customized by implementing a subclass of
+  /// [SliderTrackShape].
+  final SliderTrackShape trackShape;
+
+  /// The shape and behavior that will be used to draw the [Slider]'s ticks.
```

Comment:
Will resolve from above question:

I was following the pattern of the other existing components here. If I remove "This can be customized by implementing a subclass of Foo" from the 3 new components, should I also take it away from the existing 2?

Reply:
No need to fix them up in this PR. Filing an issue, so that the work happens eventually, would be helpful.

Reply:
Done, I ended up keeping some comments consistent by updating the, and some constsent by leaving tham as is, but I have filed https://github.com/flutter/flutter/issues/25672 to further improve the docs, and create a diagram